src/content/smart-contracts/index.md

```diff
@@ -29,47 +23,43 @@ Alice and Bob are having a bicycle race. Let's say Alice bets Bob $10 that she w
 
 This silly example illustrates the problem with any non-smart agreement. Even if the conditions of the agreement get met (i.e. you are the winner of the race), you must still trust another person to fulfill the agreement (i.e. payout on the bet).
 
-## Smart contracts {#smart-contracts}
-
-Smart contracts digitize agreements by turning the terms of an agreement into computer code that automatically executes when the contract terms are met.
-
 ### A digital vending machine {#vending-machine}
 
 A simple metaphor for a smart contract is a vending machine, which works somewhat similarly to a smart contract - specific inputs guarantee predetermined outputs.
 
 - You select a product
-- The vending machine returns the amount required to purchase the product
-- You insert the correct amount
-- The vending machine verifies you have inserted the correct amount
-- The vending machine dispenses the product of choice
+- The vending machine displays the price
+- You pay the price
+- The vending machine verifies that you paid the right amount
+- The vending machine gives you your item.
 
 The vending machine will only dispense your desired product after all requirements are met. If you don't select a product or insert enough money, the vending machine won't give out your product.
 
 ### Automatic execution {#automation}
 
-One of the most significant benefits smart contracts have over regular contracts is that the outcome is automatically executed when the contract conditions are realized. There is no need to wait for a human to execute the result. In other words: smart contracts remove the need for trust.
+The main virtue of a smart contract is that it automatically executes the outcome when its conditions are met. There is no need to wait for a human to execute the result. This removes the need for a  trusted intermediary.
 
-For example, you could write a smart contract that holds funds in escrow for a child, allowing them to withdraw funds after a specific date. If they try to withdraw the funds before the specified date, the smart contract won't execute. Or, you could write a contract that automatically gives you a digital version of a car's title when you pay the dealer.
+For example, you could write a smart contract that holds funds in escrow for a child, allowing them to withdraw funds after a specific date. If they try to withdraw before that, the smart contract won't execute. Or you could write a contract that automatically gives you a digital version of a car's title when you pay the dealer.
```

Comment:
Although not wrong, it might be a little unclear what 'that' refers to here.

```suggestion
For example, you could write a smart contract that holds funds in escrow for a child, allowing them to withdraw funds after a specific date. If they try to withdraw before that date, the smart contract won't execute. Or you could write a contract that automatically gives you a digital version of a car's title when you pay the dealer.
```